README.md

```diff
@@ -1,3 +1,58 @@
 # Pantavisor
 
-The near bare metal multi-container supervisor and OS management system.
+## What is Pantavisor? 
+Pantavisor is the easiest way to build and manage embedded Linux projects with lightweight containers. Put your Linux distribution or custom-made firmware and userland into containers and get all of the benefits of portable containerized lifecycle management without needing to replace your distribution. 
+
+Pantavisor is the Linux device init system that turns the runtime into a set of containerized microservices. It is not a container engine but rather a framework for assembling and managing containerized building blocks for firmware and applications. Pantavisor provides a simple way to deploy and atomically manage your containerized embedded firmware and apps across millions of devices in a reproducible manner. 
+
+### Meets the requirements of low-spec devices 
+To ensure it can cover the low-spec end of the market, Pantavisor brings full container functionality into the single binary that keeps its size as small as possible. Depending on the software functions that are built into the container, the size can vary, but the average size for a fully functional system puts the Pantavisor binary at around 350 Kilobytes (as compressed initial ramdisk).
```

Comment:
"into a single", "the" sounds weird to me, but maybe I'm wrong. Also, lots of "size" in this paragraph